src/studio/src/designer/frontend/dashboard/common/components/RepoList.tsx

```diff
@@ -45,11 +46,7 @@ const defaultArray: IRepository[] = [];
 
 const useStyles = makeStyles({
   repoLink: {
-    color: '#579b2e',
-
-    '&:hover': {
-      color: '#579b2e',
-    },
+    color: '#57823D',
```

Comment:
I think you need to set the same color on hover, otherwise there might be some global css styles taking over

Reply:
Sure did. Thanks, fixed.